samples/bluetooth/broadcast_audio_source/boards/nrf52840dk_nrf52840.conf

```diff
@@ -1 +1,5 @@
 CONFIG_BT_CTLR_ADV_ISO=y
+# Supports the highest SDU size required by any BAP LC3 presets (155)
+CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=155
+# Supports the highest advertising data that can be set in a single HCI command
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=251
```

Comment:
```suggestion
# Zephyr Controller tested maximum advertising data that can be set in a single HCI command
CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=191
```

Reply:
Hmm, so we don't always support The maximum length of a single HCI command? 

In the BSIM tests we actually set `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=255` I see, so isn't it tested at 251 already?

Reply:
If a periodic advertising is associated and/or a channel classification is requested the common extended header format size increases, hence controller is tested only to 191 bytes as supported AD data length max.

The `bsim_test_audio` test should reduce `CONFIG_BT_CTLR_ADV_DATA_LEN_MAX` to 191 bytes. Hope 191 is sufficient for audio usecases (otherwise controller has to support AD data fragmentation into multiple on-air PDU).